api/templates/api.mustache

```diff
@@ -358,25 +358,25 @@ func (a *{{{classname}}}Service) {{nickname}}Execute(r {{#structPrefix}}{{&class
 
 	var errorPrefix string
 	if a.isOnlyOSS {
-		errorPrefix = "InfluxDB OSS-only command failed: "
+		errorPrefix = "InfluxDB OSS-only command failed"
 	} else if a.isOnlyCloud {
-		errorPrefix = "InfluxDB Cloud-only command failed: "
+		errorPrefix = "InfluxDB Cloud-only command failed"
 	}
 
 	if localVarHTTPResponse.StatusCode >= 300 {
 		body, err := GunzipIfNeeded(localVarHTTPResponse)
 		if err != nil {
 			body.Close()
-			return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s%w", errorPrefix, err)
+			return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s: %w", errorPrefix, err)
 		}
 		localVarBody, err := _ioutil.ReadAll(body)
 		body.Close()
 		if err != nil {
-			return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s%w", errorPrefix, err)
+			return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s: %w", errorPrefix, err)
 		}
 		newErr := GenericOpenAPIError{
 			body:  localVarBody,
-			error: _fmt.Sprintf("%s%s", errorPrefix, localVarHTTPResponse.Status),
+			error: _fmt.Sprintf("%s: %s", errorPrefix, localVarHTTPResponse.Status),
```

Comment:
Slightly more clear(?):
```suggestion
			error: _fmt.Sprintf("%s: code %s", errorPrefix, localVarHTTPResponse.Status),
```

Reply:
I fixed the logic so that the status code part of the error string chain gets preserved all the way through rather than being overwritten, and the resulting error looks like this:

```
2021/06/30 11:36:33 INFO: Downloading metadata snapshot
Error: failed to backup metadata: failed to download metadata snapshot: InfluxDB OSS-only command failed: 404 Not Found: path not found
````

I hadn't been able to see the status code in any error messages before, but now that I see that it includes the both the response code and the response text, having the extra text in front of that seems redundant.